mkdocs.yml

```diff
@@ -0,0 +1,55 @@
+site_name: sparse
+repo_url: https://github.com/pydata/sparse.git
+edit_uri: edit/main/docs/
+theme:
+  name: material
+  logo: assets/images/logo.png
+  favicon: assets/images/logo.svg
+  features:
+        - toc.integrate
+        - navigation.tracking
+        - navigation.instant
+        - navigation.prune
+        - navigation.tabs
+        - navigation.footer
+markdown_extensions:
+  - tables
+  - toc:
+      toc_depth: 3
+
+plugins:
+- search
+- autorefs
+- gen-files:
+    scripts:
+    - scripts/gen_ref_pages.py
+- literate-nav:
+    nav_file: SUMMARY.md
+- section-index
+- mkdocstrings:
+    handlers:
+          python:
+            import:
+            - https://numpy.org/doc/stable/objects.inv
+            - https://docs.python.org/3/objects.inv
+            - https://docs.scipy.org/doc/scipy-1.14.0/objects.inv
```

Comment:
Does someone know where the general `objects.inv` for `scipy` is located? (to remove the hardcoded version) 
The links for `python` and `scipy` are not like the ones on `config.py` for Sphinx.

Reply:
The link is: https://docs.scipy.org/doc/scipy/objects.inv